src/fmgc/src/guidance/lnav/transitions.ts

```diff
@@ -11,11 +11,11 @@ export abstract class Transition implements Guidable {
 
     abstract getPseudoWaypointLocation(distanceBeforeTerminator: NauticalMiles): LatLongData | undefined;
 
-    abstract getNominalRollAngle(gs): Degrees;
+    abstract getNominalRollAngle(gs: number): Degrees;
```

Comment:
I have made these all `Knots`, fixing a bug in the process in an upcoming PR. This is fine though, and I will rebase once this PR lands. 👍